analyzer/src/main/kotlin/managers/GoDep.kt

```diff
@@ -28,6 +28,7 @@ import java.nio.file.Paths
 
 import org.ossreviewtoolkit.analyzer.AbstractPackageManagerFactory
 import org.ossreviewtoolkit.analyzer.PackageManager
+import org.ossreviewtoolkit.downloader.VcsHost
```

Comment:
Commit message: I might be mistaken, but I though it's anyway not directly the `go get -d` call that return the VCS information, but it only creates a local working tree that we inspect for VCS information later. Is that not the case?

In any case, could you please provide more detail under which circumstances VCS information might not be available?

Reply:
I did more research and the issue is that from Go 1.16 the behavior of `go get` changed.
I added a commit to move back to the old behavior.

For more information see: https://maelvls.dev/go111module-everywhere/

Reply:
Regarding the fallback URL.
I'm not sure if it is required anymore. In my tests all sources downloaded via `go get` contained valid VCS metadata.
What do you think?

Reply:
I'm also a bit undecided. On the one hand it's always safer to have a fallback; on the other hand, if we can't for the love of God argue under which circumstances we'd run into that fallback, it's a bit weird to add.

I'd probably be pragmatic here and say, now that we have it, we keep it. But we should make clear in a code comment that this should actually never happen, and eventually log a warning in case it does.

Reply:
I changed the code comment for the fallback code.
I did not add the log message as it would require more logic which is currently hidden in `processProjectVcs()`.

But I can also add it if you insist.

Reply:
That's finer with me, I do not insist on the warning.